docs/architecture/rewards.md

```diff
@@ -0,0 +1,25 @@
+# Staking Rewards
+
+ZetaChain is a delegated proof of stake blockchain. Holders of ZETA tokens can
+stake their tokens by delegating to validators. Staking tokens allows holders to
+participate in governance and earn rewards for securing the network.
+
+Currently, staking rewards are comprised of transaction fees collected each
+block and fixed rewards coming from the emissions pool.
+
+$$
+StakingRewards = TransactionFees + {EmissionsPool \over NumberOfBlocksPerYear × 4}
+$$
+
+Emissions pool address:
+[zeta1w43fn2ze2wyhu5hfmegr6vp52c3dgn0srdgymy](https://zetachain.blockpi.network/lcd/v1/public/cosmos/auth/v1beta1/module_accounts)
+
+You can find the emissions pool balance by querying the API:
+
+https://zetachain.blockpi.network/lcd/v1/public/cosmos/bank/v1beta1/balances/zeta1w43fn2ze2wyhu5hfmegr6vp52c3dgn0srdgymy
+
+- The emission pool is currently funded manually
+- Currently, there is no ZETA token emission in the network
```

Comment:
What does this mean? We do have emissions enabled ?

Reply:
Replaced with "Currently, there is no emission of new ZETA tokens (inflation) in the network". 86bf640

Reply:
Do you think it would be sufficient to say just 
`Currently, there is no inflation in the network` ?

Reply:
99184c3